isis/src/base/objs/ProcessImport/ProcessImport.cpp

```diff
@@ -1263,6 +1259,24 @@ namespace Isis {
         att.setPixelType(p_pixelType);
       }
     }
+  }
+
+
+  /**
+   * Create the output file. Note that all the appropiate calls to at least
+   * SetDimensions and SetPixelType should be made prior to calling this method.
+   *
+   * @param parameter The parameter name that holds the output file name.
+   *
+   * @return @b Isis::Cube Output cube.
+   *
+   * @throws Isis::iException::Message "Unsupported pixel type."
+   */
+  Isis::Cube *ProcessImport::SetOutputCube(const QString &parameter) {
+    CubeAttributeOutput &att =
+      Application::GetUserInterface().GetOutputAttribute(parameter);
+
+    SetAttributes(att);
```

Comment:
This looks fine, but I don't understand why this change was made?

Reply:
@acpaquette This change was made more-or-less to avoid changing `ProcessImport` too much. The application parameter version of `SetCube("TO")` sets up the attributes's pixel propagation and min/max values for you, but the `SetCube(filename, attributes)` does not. We need to use the second function signature now in `leisa2isis` since `ProcessImport` no longer has access to the UI, but the attributes aren't set up in this way in in this version and this resulted in an error along the lines of "you told me to propagate pixel type, but not what the type is." I moved SetAttributes out so I could call it directly from `leisa2isis` without changing the existing functionality of any of the functions inside `ProcessImport`